autoload/vebugger.vim

```diff
@@ -268,26 +270,39 @@ function! vebugger#createDebugger(command)
     return l:debugger
 endfunction
 
+if has('timers')
+    function! ReadingTimerCallback(debugger, timer_id)
```

Comment:
Are you sure about `start_timer` not accepting script local functions? Because I just checked, and this works fine in both Vim8 and Neovim:

```vim
let s:x = 0

function! s:foo(timerId)
	let s:x += 1
	echo 'foo' s:x a:timerId
endfunction

call timer_start(1000, function('s:foo'), {'repeat': 3})
```

[Neovim does not support partials yet](https://github.com/neovim/neovim/issues/5065). The only other way to pass variables together with a function is an object - but `start_timer` does not support that. Neovim will probably get partials in the near future, but I'm not willing to drop backward compatibility yet([I even gave up some encapsulation to support Vim7.3](https://github.com/idanarye/vim-vebugger/issues/5)).

At some point in the future I want to bump the major version and rewrite the message bus to use jobs. This will make the timer redundant. Until then - we'll have to use a global variable.